### PR TITLE
Merging master to 2018.3

### PR DESCRIPTION
**Pull request by haeseung81**
Merging xrt testcases from the master branch

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

File: tests/xrt/00_hello/main.cpp [hunk 1]
```diff
@@ -14,9 +14,6 @@
  * under the License.
```
Comment (rradjabi) [inline tests/xrt/00_hello/main.cpp]: Move copyright date to -2018, then it matches the file in master/2019.1

File: tests/xrt/02_simple/main.cpp [hunk 2]
```diff
@@ -14,8 +14,6 @@
  * under the License.
```
Comment (rradjabi) [inline tests/xrt/02_simple/main.cpp]: Bump copyright to -2018

**Comment by rradjabi** [comment]
@haeseung81 

1) Can you merge the changes from #847 so that these test files match more closely? 
2) In #847, I removed utils.h from inside the individual test folders, you should do the same and update the common utils.h, and have each test case include this version. I'm not familiar with the YML file, but it looks like that's what's going on.

You can discus with me before making those changes if you like.

**Pull request closed without merging**